.claude/commands/tm/add-task/add-task.md

```
@@ -0,0 +1,78 @@
+Add new tasks with intelligent parsing and context awareness.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)